libraries/Update/examples/OTAWebUpdater/OTAWebUpdater.ino

```diff
@@ -0,0 +1,102 @@
+#include <WiFi.h>
+#include <WebServer.h>
+#include <ESPmDNS.h>
+#include <Update.h>
+#include <Ticker.h>
+#include "html.h"
+
+#define SSID_FORMAT "ESP32-%06X"   // 12 chars total
+//#define PASSWORD "test123456"    // generate if remarked
+
+WebServer server(80);
+Ticker tkSecond;
+uint8_t otaDone = 0;
+
+const char* alphanum = "0123456789!@#$%^&*abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+String generatePass(uint8_t str_len) {
+  String buff;
+  for(int i = 0; i < str_len; i++)
+    buff += alphanum[random(strlen(alphanum)-1)];
+  return buff;
+}
+
+void apMode() {
+  WiFi.mode(WIFI_AP);
+  char ssid[13];
+  char passwd[11];
+  uint32_t espmac = ESP.getEfuseMac() >> 24; 
+  snprintf(ssid, 13, SSID_FORMAT, espmac);
+#ifdef PASSWORD
+  snprintf(passwd, 11, PASSWORD);
+#else
+  snprintf(passwd, 11, "%s", generatePass(10).c_str());
```

Comment:
i'm not sure why the compiler is complaining about truncation, but I suggest you make `passwd` be 16 bytes long instead.

Reply:
The size is actually fairly specific, as it allows you to do a wifi: QR code in a 64 bit square.

Reply:
I'm not getting this error. Are you using some additional flags?

Reply:
How about I just take out "%s"?